app/backend/src/tests/test_interceptors.py

```diff
@@ -110,7 +110,7 @@ def test_logging_interceptor_all_ignored():
         message = random_hex()
 
         def TestRpc(request, context):
-            context.abort(status_code, message)
+            context.abort(status_code, message)  # noqa: B023
```

Comment:
Online I see: 'B023 	[function-uses-loop-variable](https://docs.astral.sh/ruff/rules/function-uses-loop-variable/) 	Function definition does not bind loop variable {name}"

This seems to be the incorrect noqa?

Reply:
No, this is the correct noqa - it complains because we define a class with loop iterables that will mutate in the next iteration. Here, it does not matter as everything happens within that iteration and then the class is defined again, so I added the noqa tag.

Reply:
Okay, makes sense!